src/util.cpp

```diff
@@ -625,6 +630,15 @@ boost::filesystem::path GetConfigFile()
     return pathConfigFile;
 }
 
+boost::filesystem::path GetMcCryptoConfigFile()
+{
+    boost::filesystem::path pathConfigFile(GetArg("-mc_crypto_conf", "mc_crypto_log_config.yaml"));
```

Comment:
Can be worth adding description of new param `"-mc_crypto_conf"` (and maybe also `"-conf"`) in docs at function `HelpMessage` of file `init.cpp`?

Reply:
Yeah, makes sense. Done. Notice that `-conf` was already documented there.